config/packages/nelmio_api_doc.yaml

```diff
@@ -9,4 +9,4 @@ nelmio_api_doc:
                     type: basic
     areas: # to filter documented areas
         path_patterns:
-            - ^/v1/api(?!/doc$) # Accepts routes under /api except /api/doc
+            - ^/api/v1(?!/doc$) # Accepts routes under /api except /api/doc
```

Comment:
This is a significant API change, no?

Reply:
Yes, but that's the api docs only. For the real api entry points there is a fallback to the existing url see bellow.